sr_utilities_common/scripts/sr_utilities_common/conditional_delayed_rostool.py

```diff
@@ -86,9 +86,12 @@ def wait_for_conditions(conditions_to_satisfy, timeout):
 
     package_name = rospy.get_param("~package_name")
     executable_name = rospy.get_param("~executable_name")
-    arguments_list = rospy.get_param("~launch_args_list")
     timeout = rospy.get_param("~timeout", 0)
 
+    if rospy.has_param("~launch_args_list"):
```

Comment:
there is a much cleaner way rospy.get_param("~launch_args_list", "")

Reply:
ahhhh thank you, that's so much nicer!

Reply:
https://github.com/shadow-robot/common_resources/pull/149/commits/340c00ffd3dbf1633d3615a41d7bb2b4ccdf26c0